### PR TITLE
Adding 'The Question' section and starting on 'The Concept'.

### PR DESCRIPTION
**Pull request by peytonchance**
This short PR adds 'The Question' and the beginning of 'The Concept' section. Reapproaching media queries has been a major lightbulb moment.

image
image

### CODE REVIEW

File: src/html/components/concept.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div class="concept" id="concept">
```
Comment (jeremyfrank) [inline src/html/components/concept.html]: Another thing you could do with the markup is to use `<section>` elements for major parts of the page, like for each of these main page blocks. It's like a div, but adds a bit of semantic meaning to the structure.

File: src/html/components/question.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="question" id="question">
+  <h2 class="section-title">The Question</h2>
+  <p class="section-intro">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</p>
+  <img src="images/phone.svg" alt="" class="phone">
+  <p>Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<sup><a href="">4</a></sup> Smartphones offer many additional features but impose a barrier of entry that seemed too high. Reliance on personal computers is also problematic, especially among disadvantaged patients.</p>
+  <p>We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<sup><a href="">5</a></sup> Furthermore, adherence studies using text reminders show promising results.<sup><a href="">6</a></sup>  Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<sup><a href="">7</a></sup></p>
```
Comment (jeremyfrank) [inline src/html/components/question.html]: One pattern we like to do for things that have yet to be completed is to add a `TODO` note. In the case of this anchor (`<a href="">`) that would be `<a href="#TODO">`. That way you can later do search for "TODO" to catch anything you might have forgot to complete.

Other examples would be regular html or css comments:
```
<!-- TODO: wire-up to database -->
/* TODO add a fallback */
```

File: src/html/index.html [hunk 3]
```diff
@@ -1,10 +1,12 @@
 {% extends 'layouts/application.html' %}
 
 {% block content %}
-<div class="top-fold">
+<main>
```
Comment (jeremyfrank) [inline src/html/index.html:4]: 👍 Good call.

File: src/stylesheets/components/_concept.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-top: 180px;
+  padding-top: 130px;
+  position: relative;
+  width: 100%;
+  
+  .section-title {
+  	color: $white;
+  	margin-bottom:-10px;
```
Comment (jeremyfrank) [inline src/stylesheets/components/_concept.scss]: Indents are off here (too much). I wonder if stylelint isn't running? Might need to add it as an option to your blendid config. I'll look into it some and see what I can find for you.

File: src/stylesheets/components/_concept.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  .section-intro {
+  	color: $white;
+  	font-size: 2.5em;
+  }
+
+  p {
+    margin: 10px auto 30px auto;
```
Comment (jeremyfrank) [inline src/stylesheets/components/_concept.scss]: You could remove the last `auto` here, as it'll be repeated from the other instance, e.g. `margin: 10px auto 30px;`

File: src/stylesheets/components/_concept.scss [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+      width: 60%;
+    }
+  }
+}
+
+.florence {
+ margin-left: 18px;
+ position: absolute;
+ width: 250px;
+ top: -150px;
```
Comment (jeremyfrank) [inline src/stylesheets/components/_concept.scss]: Indent these 4 declarations.

**Review by jeremyfrank — COMMENTED** [review]
There are some spacing oddities (mixed tabs/spaces perhaps?) that could be fixed with editorconfig.

Here's a sample from a recent project that should do this for you:
```
# http://EditorConfig.org

root = true

# Unix-style newlines with a newline ending every file
[*]
end_of_line = lf
trim_trailing_whitespace = true
insert_final_newline = true
indent_style = space
indent_size = 2

[*.md]
trim_trailing_whitespace = false
```

File: src/stylesheets/components/_concept.scss (hunk 4 above)
Comment (jeremyfrank) [reply src/stylesheets/components/_concept.scss]: According to the stylelint docs, there are a 5 different ways to use it, 2 of which are:
* via a plugin for your build tool
  * Digging into blendid a little bit, I don't think it currently supports passing additional plugins directly to the stylesheets task. So you'd have to write a new css processing task (including the gulp-stylelint plugin) and pass it in the task config in the `alternateTask` option. This doesn't sound great, so the next best options is...
* via a plugin for your text editor
  * Just install http://www.sublimelinter.com/en/stable/ and have it work directly in your editor. At first glance it looks tricky to set up, but I've never used it so I can't vouch for it.

Comment (peytonchance) [reply src/stylesheets/components/_concept.scss]: Ah, this is because I switched to Sublime from Atom and didn't do that install yet - on it now.

Comment (jeremyfrank) [reply src/stylesheets/components/_concept.scss]: The docs I referenced: https://github.com/stylelint/stylelint#getting-started

**Pull request merged**